metron-platform/metron-parsers/src/test/java/org/apache/metron/parsers/integration/SquidIntegrationTest.java

```diff
@@ -29,6 +29,11 @@ String getSensorType() {
   }
 
   @Override
```

Comment:
We could spin up a new integration test, but I thought this did the trick and didn't add the cost of another integration test to the build time.  

I do have a test ensuring the value gets passed through to the parser.  This just completed the loop.  If we don't feel that an integration test is necessary here, then I can remove it too.